blueocean-plugin/src/test/java/io/jenkins/blueocean/jsextensions/JenkinsJSExtensionsTest.java

```diff
@@ -56,7 +56,7 @@ public void test() {
             } else if ("blueocean-personalization".equals(pluginId)) {
                 Assert.assertEquals(1, extensionPoints.size());
                 Assert.assertEquals("components/ActionLink", extensionPoints.get(0).get("component"));
-                Assert.assertEquals("jenkins.dashboard.item.action", extensionPoints.get(0).get("extensionPoint"));
+                Assert.assertEquals("jenkins.pipeline.list.action", extensionPoints.get(0).get("extensionPoint"));
```

Comment:
only 🐜  I have about these extension points (and the existing ones) is `jenkins.` is pretty unnecessary, tbh... we might want to revisit this before a non-alpha release